clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/predicate/DoubleGreaterThanPredicate.java

```diff
@@ -0,0 +1,14 @@
+package com.linkedin.venice.client.store.predicate;
+
+public class DoubleGreaterThanPredicate implements DoublePredicate {
+  private final double threshold;
+
+  DoubleGreaterThanPredicate(double threshold) {
+    this.threshold = threshold;
+  }
+
+  @Override
+  public boolean evaluate(double value) {
+    return value > threshold;
```

Comment:
There might be some issue here, as when run this check, we could encounter this     
```
    double a = 0.1d + 0.2d; // 0.30000000000000004
    double b = 0.3d;
    //a > b: true 
    // a - b: 5.551115123125783E-17
```


When we check with epsilon, we may encounter 

```
DoubleGreaterThanPredicate doubleGreaterThanPredicate = new DoubleEqualsPredicate(a);
doubleGreaterThanPredicate.evaluate(b); // True

DoubleEqualsPredicate doubleEqualsPredicate = new DoubleEqualsPredicate(a, 1e-15);
doubleEqualsPredicate.evaluate(b) // True
```

Should we add epsilon to greater or lower check to ensure mutual-exclusive?